lib/node_modules/@stdlib/random/base/mt19937/test/test.factory.js

```diff
@@ -0,0 +1,267 @@
+		t.equal( isPositiveInteger( v ), true, 'returns a positive integer' );
+		t.equal( v >= 1 && v <= UINT32_MAX, true, 'returns an integer between 1 and 2^32-1 (inclusive)' );
+	}
+	t.end();
+});
+
+tape( 'the function returns a seeded pseudorandom number generator', function test( t ) {
+	var mt199371;
+	var mt199372;
+	var seed;
+	var v1;
+	var v2;
+	var i;
+
+	seed = round( Date.now() / 1000 );
+
+	mt199371 = factory( seed );
+	mt199372 = factory( seed );
+
+	t.notEqual( mt199371, mt199372, 'separate generators' );
+
+	for ( i = 0; i < 1e3; i++ ) {
+		v1 = mt199371();
+		v2 = mt199372();
+		t.equal( v1, v2, 'both return same number' );
+	}
+	t.end();
+});
+
+tape( 'attached to the returned function is the generator name', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.NAME, 'mt19937', 'has `NAME` property' );
+	t.end();
+});
+
+tape( 'attached to the returned function is the generator seed', function test( t ) {
+	var mt19937 = factory( 12345 );
+	t.equal( typeof mt19937.SEED, 'number', 'has `SEED` property' );
+	t.equal( mt19937.SEED, 12345, 'equal to provided seed' );
+	t.end();
+});
+
+tape( 'attached to the returned function is the minimum possible generated number', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.MIN, 1, 'has `MIN` property' );
+	t.end();
+});
+
+tape( 'attached to the returned function is the maximum possible generated number', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.MAX, UINT32_MAX, 'has `MAX` property' );
+	t.end();
+});
+
+tape( 'attached to the returned function is a `normalized` method for generating pseudorandom numbers strictly between 0 (inclusive) and 1 (exclusive)', function test( t ) {
+	var mt19937;
+	var v;
+	var i;
+
+	mt19937 = factory();
+	for ( i = 0; i < 1e4; i++ ) {
+		v = mt19937.normalized();
+		t.equal( typeof v, 'number', 'returns a number' );
+		t.equal( v >= 0.0 && v < 1.0, true, 'returns a number between 0 (inclusive) and 1 (exclusive)' );
+	}
+	t.end();
+});
+
+tape( 'attached to the `normalized` method is the generator name', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.normalized.NAME, 'mt19937', 'has `NAME` property' );
+	t.end();
+});
+
+tape( 'attached to the `normalized` method is the generator seed', function test( t ) {
+	var mt19937 = factory( 12345 );
+	t.equal( typeof mt19937.normalized.SEED, 'number', 'has `SEED` property' );
+	t.equal( mt19937.normalized.SEED, 12345, 'equal to provided seed' );
+	t.end();
+});
+
+tape( 'attached to the `normalized` method is the minimum possible generated number', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.normalized.MIN, 0.0, 'has `MIN` property' );
+	t.end();
+});
+
+tape( 'attached to the `normalized` method is the maximum possible generated number', function test( t ) {
+	var mt19937 = factory();
+	t.equal( mt19937.normalized.MAX, (UINT32_MAX-1.0)/(UINT32_MAX), 'has `MAX` property' );
```

Comment:
I don't believe this is correct. We now use the 53 bits of precision algo, not the 32 bits of precision algo.